spec/models/game_spec.rb

```diff
@@ -43,4 +43,9 @@
       expect(game.white_to_move).to eq(true)
     end
   end
+
+  it "should have correct number of pieces" do
+      game = Game.create
+      game.pieces.count==32
```

Comment:
That was probably the test legitimately failing. It may have been because the line just before it doesn't actually create a valid game and save it to the DB or for some other reason.